README.md

```diff
@@ -76,8 +76,14 @@ It's based on Rails 4 and Ruby 2.2.2.
 Some gems have native extensions.
 You should have GCC installed on your development machine.
 
-* `brew install qt` - to run specs with [Capybara Webkit](https://github.com/thoughtbot/capybara-webkit)
-* `brew install phantomjs198` - to run Javascript unit tests
+* `qt` - to run specs with [Capybara Webkit](https://github.com/thoughtbot/capybara-webkit)
+* `phantomjs198` - to run Javascript unit tests
+
+You can setup requried dependencies from `Brewfile`:
```

Comment:
I guess, there is no need in using "can" here since we're enforcing this approach:

> Setup required dependencies from `Brewfile`: